modules/busco/main.nf

```diff
@@ -0,0 +1,42 @@
+process BUSCO {
+    tag "$meta.id"
+    label 'process_medium'
+    
+    conda (params.enable_conda ? "bioconda::busco=5.2.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/busco:5.2.2--pyhdfd78af_0':
+        'quay.io/biocontainers/busco:5.2.2--pyhdfd78af_0' }"
```

Comment:
No they don't include the lineage datasets. The datasets can be pre-downloaded separately and provided as an input, or it will be downloaded automatically during execution.